index.js

```diff
@@ -22,7 +22,25 @@ function getPercentageScore(score) {}
  * @param {object} score2 - An object representing another score of an assignment. Has the same shape as `score1`.
  * @returns {number} A rounded integer, representing the highest percentage scored among the two given scores.
  */
-function getHighestOfTwoScores(score1, score2) {}
+function getHighestOfTwoScores(score1, score2) {
+  const score = getPercentageScore
```

Comment:
It's _probably_ better to write `getPercentageScore` every time when it costs a line to rename the variable.

`getPercentageScore`, while longer, is also a much clearer name. It's not obvious from reading this code that `score` is a function while `score1` and `score2` are not. It's the difference between your code "scoring" those scores and "getting the percentage score" from those scores.

It' s worth the extra typing to have good variable names. And it's not even extra typing if you're using auto-complete—in fact, it's less, as you could type "ge" and hit tab or enter when auto-complete suggests the function name!